src/GovUk.Education.ExploreEducationStatistics.Data.Model/Services/FootnoteRepository.cs

```diff
@@ -183,6 +183,32 @@ private IQueryable<Footnote> GetBaseFootnoteQuery()
                 .Include(footnote => footnote.Subjects);
         }
 
+        public async Task<IList<Subject>> GetSubjectsWithNoFootnotes(Guid releaseId)
+        {
+            return await _context.ReleaseSubject
```

Comment:
Not quite! It's working on `ReleaseSubject` instead of `Footnote`. Don't think we do this kind of specific query between subjects-foonotes anywhere else. I was in a bit of quandry over whether to put it in here or `SubjectService`, but decided on here because it was relevant to footnotes still.

I think the resultant query isn't _too_ bad, at least compared to the main `GetFootnotes` query which is pretty nasty